streams-contrib/streams-provider-twitter/src/main/jsonschema/com/twitter/tweet.json

```diff
@@ -1,10 +1,7 @@
 {
+    "type": "object",
```

Comment:
I think some of these changes might be duplicated in: https://github.com/w2ogroup/incubator-streams/pull/98 May be a good idea to merge that branch into this one and ensure there are no conflicts/missed schema changes